snap/hooks/install

```diff
@@ -10,6 +10,9 @@ snapctl set infra-if=wlan0
 snapctl set thread-if=wpan0
 snapctl set radio-url="spinel+hdlc+uart:///dev/ttyACM0"
 snapctl set autostart=false
+snapctl set interface-name=wpan0
+snapctl set listen-address="0.0.0.0"
```

Comment:
`0.0.0.0` is IPv4 only. Please leave the default as `::` as documented on #15.

See also: https://github.com/openthread/ot-br-posix/blob/e66e1bb357c28f476ec854dfd465b070d0249ca2/src/web/main.cpp#L50

Reply:
Updated